x-pack/plugins/security_solution/public/resolver/view/graph_controls.tsx

```diff
@@ -68,8 +106,11 @@ export const GraphControls = React.memo(
     const dispatch: (action: ResolverAction) => unknown = useDispatch();
     const scalingFactor = useSelector(selectors.scalingFactor);
     const { timestamp } = useContext(SideEffectContext);
+    const [activePopover, setActivePopover] = useState<null | 'schemaInfo' | 'nodesLegend'>(null);
```

Comment:
SchemaInfo is the schema information popover
NodesLegend is the legend of the node types in the popover

I think that makes sense re what would cause it to close, but the points made in the call made it seem like users may want to keep it open while navigating

So maybe only the refresh button?